src/DataTable/ExpandRow.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Add, Remove } from '../../icons';
+
+const ExpandRow = ({ row }) => (
+  <span {...row.getToggleRowExpandedProps()}>
+    {row.isExpanded ? <Remove /> : <Add />}
+  </span>
+);
+
+ExpandRow.propTypes = {
+  /** Row data that is received from `react-table` API. */
+  row: PropTypes.object.isRequired, // eslint-disable-line
```

Comment:
nit: typically we'd write this as `PropTypes.shape({})` to define an object prop type without really caring what's in it.

Reply:
Updated, thanks! For some reason I forgot `PropTypes.shape` existed 😅